catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -158,7 +158,11 @@ public MetacardImpl(Metacard metacard, MetacardType type) {
         }
         map = new HashMap<>();
         for (AttributeDescriptor attribute : metacard.getMetacardType().getAttributeDescriptors()) {
```

Comment:
Not mandatory but renaming this to `descriptor` instead of `attribute` would really help with clarity. We throw the word _"attribute"_ around so much without specificity that it's hard to follow precisely which component we're referring to. 

Not part of your change-set so completely optional, I would say.